### DISCUSSION TITLE
Positive feedback.

### DISCUSSION THREAD

**Discussion by ljonesfl** [tsightler/ring-mqtt#930] [General Discussion]
Throughout my many many years in the tech industry, it's very rare that I encounter something this technical in nature that simply works. I was shocked to find that after installing and with zero configuration besides logging in, all of my plethora of devices were discovered with zero issue.
Well done everyone.

**Comment by tsightler** [comment]
Thanks @ljonesfl for the kind words.  I've worked hard from the start to make it plug-and-play as this project has like 10,000 users these days and I don't have time to help people individually, so it needs to just work.

What's funny is, about 90% of the issues people do report, especially new users, is that they think it needs some kind of configuration.  So many times a user reports issues and they are trying to manually configure everything and I have to talk them into uninstalling everything and going through the entire Mosquitto/MQTT config on HA and ring-mqtt without touching anything and then they are surprised that it works!  

A lot of this credit has to go to Home Assistant, which provides a very nice internal API for addon developers to query available services and internal credentials which makes auto-magic configuration super easy while still being secure (I would say that automatic configuration is more secure than manually configured users).
